portality/settings.py

```diff
@@ -683,8 +683,16 @@
         "suggestion" : {
             "auth" : True,
             "role" : "admin",
+            "query_filters" : ["not_update_request"],
             "dao" : "portality.models.Suggestion"    # ~~->Application:Model~~
         },
+        # ~~->AdminUpdateRequestQuery:Endpoint~~
+        "update_requests": {
+            "auth": True,
+            "role": "admin",
+            "query_filters" : ["update_request"],
+            "dao": "portality.models.Suggestion"  # ~~->Application:Model~~
```

Comment:
Since this is new code, can we use the new model, rather than the back-compat model.  `portality.models.Application`